traveller/modules/box__default/auth/models.py

```diff
@@ -82,6 +82,7 @@ class User(UserMixin, PkModel):
     is_email_confirmed = db.Column(db.Boolean(), nullable=False, default=False)
     email_confirm_date = db.Column(db.DateTime)
     bio = db.Column(db.String(200))
+    image = db.Column(db.String(128))
 
```

Comment:
@0x15F9 can you add a get_profile_image_url) method in the model returning the url of the model?